app/presenters/menu/default_menu.rb

```diff
@@ -213,7 +213,7 @@ def control_menu_section
       end
 
       def automate_menu_section
-        generic_object_item = if VMDB::Config.new("vmdb").config[:product][:generic_object] == true
+        generic_object_item = if ::Settngs.product.generic_object == true
```

Comment:
There is a type on this line - "::Settngs" should be "::Settings".

Otherwise LGTM.